### PR TITLE
add comments for the next steps

### PR DESCRIPTION
**Pull request by Verenasworld**

### CODE REVIEW

File: src/app/cards/cards.component.ts [hunk 1]
```diff
@@ -31,18 +31,35 @@ ready(){
...
+
+//show overlay game over & button restart 
+}
 
+
+timer(){
+// from 100 to 0 counter -
 }
 
+win(){
```
Comment (LeoAvramVie) [inline src/app/cards/cards.component.ts:57]: wtf

**Review by LeoAvramVie — COMMENTED** [review]
see my comments

**Comment by Verenasworld** [comment]
this is realy important

**Review by LeoAvramVie — APPROVED** [review]
well done

**Pull request merged**